### ISSUE TITLE
Idea: Global Awareness Hackathon to boost project, jumpstart crowdsourcing

### ISSUE THREAD

**Issue by aschrijver** [humanetech-community/humanetech-community-awareness#13]
## Background

There is really a lot to do to get this project roaring ahead. Besides all the work that goes into setting up campaigns, produce video's and other deliverables, we need to determine strategies, organize funding initiatives, rally more people to the cause, etc. etc.

On the forum we already have a campaign team of 11 members only 2 days after the [Call for Participation](https://community.humanetech.com/t/call-for-participation-the-humane-tech-awareness-campaigns-project-has-started/2782) was posted, so that is great. But we can speed this up by launching a campaign to bootstrap this project itself.

Furthermore, while we envision professional studio's and actors to take part in creation of our promotion video's, this requires great levels of organization and possibly comes with obligations (to sponsors, etc.), that I don't think our community - until now mostly active on the community forum - can handle right now.

Also I am convinced that professional video's can be produced at very low cost using crowdsourcing. Our community contains many experts in a broad range of fields, we all have networks, and there is a whole world of people out there that are worried about tech, that should be willing to participate in fun and value-adding projects.

Lastly, the previous point, and the fact that crowdsourcing really aligns with Humane Tech concepts - using the internet to bring people together and reach great creative heights - would be a great Proof of Concept for our community and community building in general.

The idea I want to discuss here, is to use our existing Meetup structure (20+ groups all around the world) and organize a global hackathon were we work on the Awareness Campaigns project.

## Global Awareness Hackathon

This idea starts by creating a campaign in this project, and working on it like on any other campaign.

We will pick a proper date, a couple of months from now, and start preparing a hackathon event that involves most, if not all of our Meetup groups. The term 'Hackathon' is very common in tech realms, but increasingly also used in other fields, such as Marketing. During the hackathon - in our case happening on a single day - multiple teams cooperate on a variety of topics of their own choosing aligned to a common theme. In this case the theme is _Humane Tech Awareness Campaigns_,

Everything (and more) related to this project can be picked up by a team as topics at the hackathon:

- Creating video's, design artwork
- Social media promotion campaigns
- Elaborating strategies
- Executing entire campaigns
- Organize crowdfunding
- Creating stories, scripts, etc.
- _Whatever teams can dream up (flash mobs, stunts, press events, art ...)_

 Example:

> One Meetup group could select to execute the [Reality Shock](campaigns/reality-shock/README.md) campaign and produce the [Memories of a Bartender](https://github.com/humanetech-community/humanetech-community-awareness/issues/11) video deliverable.
>
> They spread posters and flyers throughout the city, that were designed by another group, and the community campaign team.
>
> As people (members and non-members) sign up for the Hackathon, they get to choose team membership of their liking, and help with preparation (voluntarily, not required).
> The Meetup group arranges one or more preparatory meetups to brainstorm for and assign tasks, so everything is ready when the Hackathon starts
>
> They manage to find a proper location - an authentic pub with sufficient space - where the video can be shot, find participants that can bring video equipment. The video will be created during the hackathon, but in the afternoon, and everyone can be a figurant in the video. Members are asked to bring clothing, if they can, for the various time settings. The art academy in their city can also help with this, for free.
>
> The pub owner is providing food and drinks. The local press is invited, and she will get free publicity in return.
>
> [ ... etcetera, well, you get the idea ... ]

## Concluding

I think such a hackathon will be great fun, highly creative, intensely social, worthwhile being part of, and it will create much publicity for our community, CHT and Humane Tech in general, drawing in many new members.

If successful we can hold it yearly, or even twice a year.

**Comment by georgeshk** [comment]
> Also I am convinced that professional video's can be produced at very low cost using crowdsourcing. Our community contains many experts in a broad range of fields, we all have networks, and there is a whole world of people out there that are worried about tech, that should be willing to participate in fun and value-adding projects.

Sorry but I strongly disagree on this point.

First, logistically, it is quite improbable we can find large teams to undertake the production of deliverables within the community or close friends, with the availability required to deliver. Not to mention members are based in different locations.

Second, my objective has always been to create videos of very high quality to maximize effectiveness (both viral effect and impact). This can only be achieved with professional studios. Remember that the feel and impact of a video depends largely on adding cinematic effects and we are clearly not equipped to come up with all the right idea. Let us not reinvent the wheel.

Third, if we proceed as I propose, we can go very fast and only need the community to act as a project management team, sourcing the various parts to make it happen. Your proposal here adds a tremendous amount of complexity and relies on the satisfaction of many conditions.

Only issue with my proposed approach is funding. But I dont think it is an impossible challenge. Much easier to ask our friends and contacts to lead many to donate to our crowfunding campaign, than ask them to organize video production teams.

**Comment by georgeshk** [comment]
Here are the very few and simple steps I would propose to take towards quickly achieving our objectives.

1. Community discusses and finalizes campaign themes, campaigns and deliverables (once one deliverable's concept is finalized, we can proceed to the next step).

2. Community approaches video studios with themes and concepts of deliverables. I will shortly draft a cover letter that anyone could send to potential video studios (we will ask for quotes and proposals, and evoke the possibility to partner with them if they give discounts or produce for free).

3. Community selects a studio for the deliverable. Costs are budgeted and we launch a crowdfunding campaign (see related issue for discussions on crowdfunding).

4. When the crowdfunding effort is successful, we commission the selected studio and manage the production.

5. Once the final product is ready and validated by the community, we start distribution through all available platforms and networks.

This fast, effective approach keeps everyone motivated and engaged in high-level activities, and leads to the production of professional, high-impact videos.

**Comment by georgeshk** [comment]
Another important comment on the practicalities: the large majority of contributors of any community tends to engage in discussions of ideas, comments and opinions. Very few have the drive and availability to take any part in the execution, hence it makes a lot of sense to consider this platform as a project management team.

**Comment by aschrijver** [comment]
> Sorry but I strongly disagree on this point.

This is okay. I am not suggesting that _all_ video's should be produced like this. Nor am I saying that all should be produced professionally. Both are valid approaches, and we can decide on a case-by-case basis. Both the professional and crowdsourced paths are valuable tools in our toolbox, and should be further developed.

> First, logistically, it is quite improbable we can find large teams [..]

Not saying it is easy, and it is improbable _at the moment_ since we only recently started being more than a forum. But you underestimate the power of crowdsourcing, I think. My computer runs completely on open-source software. OSS is a form of crowdsourcing. This is in different realm, but there are many forms of crowdsourcing that have been effective. There are exemplars.

> Second, my objective has always been to create videos of very high quality to maximize effectiveness (both viral effect and impact).

As said, I am not in any way against following this path, I want to have professionality and quality just as much as you. Note though, that most viral video's I have seen were produced by non-professionals. Just saying :)

> Third, if we proceed as I propose, we can go very fast and only need the community to act as a project management team [..]  Your proposal here adds a tremendous amount of complexity [..]

That is just great, and I highly commend to proceed as such. And regarding complexity: not really, or at least not more than a lot of other things we stil need to figure out, like marketing a video so that it goes viral. Please note that: 

- Tthis project isn ongoing and will run for years, as long as there is interest
- There is no limit to the amount of people that can participate.
- There is no time limit or deadline to any of the ideas proposed, unless we set them
- There is no restriction to the amount of ideas that can be posed
- There is no need to pick up an idea. It can wait until someone finds it interesting enough, or archived

> Here are the very few and simple steps I would propose to take towards quickly achieving our objectives.

Super! It is wonderful what you are doing in this regard :heart: 
I saw you already created an issue for the RFP. I recommend creating a high-level issue with the list of steps (you can format them as an issue checklist, I'll PM you how to do that), and create separate issues for each step. Break down further if tasks can be divided. We can then assign and create a task board (called Project in GH terminology).

> What you are suggesting is to create a campaign to raise awareness about our campaigns to raise awareness

No. There is no difference. I will create a new theme, called 'Awareness Hackathons'. It will have campaigns to organize concrete events, where the deliverables will be Meetups. Meetups are open to anyone, so the audience are worried people about tech, that want to roll up their sleeves and contribute their 2 cts, and who like social face-to-face activities.

During the hackathons they work on topics that create awareness in themselves, yes. That's just fine and still very much in line with the purpose for this project :)

**Comment by georgeshk** [comment]
Got it. I propose we try both approaches (in-house or professional production) on a case by case basis, with the final choice being made by the particular deliverable/video concept originator. I think it would make it easy to move forward, as the originator is the best placed to consider how his vision can be effectively materialized. So, I guess all good here.

> I will create a new theme, called 'Awareness Hackathons'. It will have campaigns to organize concrete events, where the deliverables will be Meetups. Meetups are open to anyone, so the audience are worried people about tech, that want to roll up their sleeves and contribute their 2 cts, and who like social face-to-face activities.

I guess I misunderstood your objectives. If the whole idea here is for members to organize hackathons on particular topics relevant to our initative (such as discussing and working on producing the Bartender video), I am all in.

**Comment by georgeshk** [comment]
Re. project and high-level issue, I am afraid I am a bit lost. The 5-step path I propose could make way I guess for a project working plan (no idea though how to work that out in Github, thanks in advance for your guidance).

**Comment by aschrijver** [comment]
> With the final choice being made by the particular deliverable/video concept originator

I agree the originator will have a big, if not biggest say, but this also depends on case-by-case basis. Some people just want to provide the script and not involved further. Other times it may be a more democratic or organic decision (If for instance you don't get the team for your approach). That's the nature of crowdsourcing. We don't set it as a fixed rule (at project scope, but per campaign/deliverable it is the strategy that leads).

> Re. project and high-level issue, I am afraid I am a bit lost.

I'll help you. It's basically a divide and conquer thing, making large chunks more manageable by breaking into smaller pieces.

But let's get all this discussion off this thread and in separate issues. This is about the Hackathon idea, and, as you see on Github too, threads can become easily TL;DR distracting from the title / topic

**Comment by georgeshk** [comment]
> Other times it may be a more democratic or organic decision 

Let us be careful here. Most people are very happy to judge, give opinions and criticize (it doesn't take much effort), but not actively involved in the heavy duty work. My view: if someone comes up with a concept and is ready to lead it to conclusion, let's give him control over the whole process and final decision-making. If someone else comes up and seems as ready to make his initiative a success, the originator will recognize that. Let us not confuse feedback with directions. I know I sound undemocratic here but I feel you should earn the right to demand major changes in direction. At the end of the day, this should be a natural process.

**Comment by georgeshk** [comment]
> You may want to consider approaching university programs such as the Academy of Creative Media, which is highly regarded in the film industry.

What a wonderful idea! Academies could provide a wonderful in-between solution between in-house and professional production.

@patmatsu : would you consider taking the lead on this very promising option?

**Comment by georgeshk** [comment]
> Let me talk to my friend Noah about this idea, and I will get back to you in a day or two, OK?

Great! Your discussions could help form a whole new strategy aimed at acadamies either in the US and Europe who could be inspired to take on our concepts.

**Comment by aschrijver** [comment]
> You may want to consider approaching university programs such as the [Academy of Creative Media](http://acm.hawaii.edu/), which is highly regarded in the film industry.

Yes, this is a great idea, and also what I was envisioning when writing in the example _"The art academy in their city can also help with this, for free."_. Crowdsourcing does not only involve the crowd that signed up, but could include many external parties on one-time, or continuous basis.

Examples:

- Art academies do a video project for CHT as part of one of their curriculum projects
- Business school students do a CHT project, a social marketing campaign, for their Marketing class
- IT students create a humane tech website learning to combine internet skills with good tech ethics
- Theatre/Movie academy provides figurants and actors. Great opportunity to build a portfolio
- Graphics design students learn to create artwork with impact, that serves a good cause too
- etcetera..

**Comment by georgeshk** [comment]
I think it is a fantastic idea worth actively exploring. Each of us in our respective countries could touch base with their national/local visual art academies and introduce our initiatives (happy to draft a template letter/email for all of us). 

At first, we would introduce our campaign objectives and seek feedback. We can them follow on with practical video projects to work on. Maybe one or two academies will feel so inspired by our efforts that they will commit to producing more than one video. Let's be flexible.

Let us see what @patmatsu comes back with. This will give us better ideas on how to explore this promising venue.

**Comment by georgeshk** [comment]
Just created a new issue (#15) on video production. For clarity, let us move there for discussions on the visual arts academies initiative.
